src/txn/txn_recover.c

```diff
@@ -430,6 +433,7 @@ __recovery_setup_file(WT_RECOVERY *r, const char *uri, const char *config)
           uri, r->files[fileid].uri, fileid);
     WT_RET(__wt_strdup(r->session, uri, &r->files[fileid].uri));
     WT_RET(__wt_config_getones(r->session, config, "checkpoint_lsn", &cval));
+
     /* If there is checkpoint logged for the file, apply everything. */
```

Comment:
I know you didn't change this line but this comment is wrong. Could you please fix the comment to add `no` before `checkpoint`:
If there is no checkpoint logged...